stdlib/string.mli

```diff
@@ -260,7 +260,39 @@ val escaped : string -> string
     [escaped s] fails).
 
     @raise Invalid_argument if the result is longer than
-    {!Sys.max_string_length} bytes. *)
+    {!Sys.max_string_length} bytes.
+
+    @deprecated Use {!String.escape_ascii} instead. *)
+
```

Comment:
We don't deprecate a function until its replacement has been available for several releases.

Reply:
Good to know, thanks! I'll remove the deprecation flag.